### ISSUE TITLE
Issues with the HMDB IDs of metabolites

### ISSUE THREAD

**Issue by orsiliska** [SysBioChalmers/Human-GEM#434]
### Description of the issue:
There are two issues with the HMDB IDs of the metabolites in the model.
1: One metabolite, 	MAM03055r (trioxilin A3) has a CHEBI ID annotated as its HMDB ID (even though the metabolite is included in HMDB as well: HMDB0001977).

2: The HMDB IDs used in the model don't all come from the updated version of HMDB. HMDB made its IDs longer with version 4 (the current versions is 5). In many cases the model contains the old IDs, which are formatted as "HMDB" followed by a 5-digit number. In the newer version this was changed to "HMDB" followed by a 7-digit number. Currently this mix of versions makes it hard to consistently connect to other datasets using HMDB IDs. 

#### Expected feature/value/output:
HMDB IDs should be fixed to all be in the format of "HMDB" followed by a 7-digit number. This shouldn't be a problem, as you'd  simply need to insert two 0s in front of the 5-digit number in the ID for metabolites that are still annotated with the old version.

```

**I hereby confirm that I have:**
- [ ] Tested my code on my own computer for running the model
- [ ] Done this analysis in the `main` branch of the repository
- [X ] Checked that a similar issue does not exist already

**Comment by haowang-bioinfo** [comment]
@orsiliska please review the [changes](https://github.com/SysBioChalmers/Human-GEM/pull/435/files) made according to your suggestions

**Comment by haowang-bioinfo** [comment]
fixed in #435
